docs/learn/learn-comparison-ethereum-2.md

```diff
@@ -106,83 +104,79 @@ Ethereum operates as a single, homogeneous chain. Each Ethereum node is divided
-    [erasure coding](./learn-parachains-protocol.md#erasure-codes) to all validators in the system,
-    such that anyone - not only the shard's validators - can reconstruct a parachain's block and
-    test its validity. The random parachain-validator assignments and secondary checks are performed
-    by randomly selected validators making it less likely for the small set of validators on each
-    parachain to collude.
-
-### Staking Mechanics
+1. {{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} finality protocol, GRANDPA,
+   finalizes batches of blocks based on
+   [availability and validity checks](./learn-parachains-protocol.md#availability-and-unavailability-phase)
+   that happen as the proposed chain grows. The time to finality varies with the number of checks
+   that need to be performed (and invalidity reports cause the protocol to require extra checks).
+   The expected time to finality is 12-60 seconds.
+
+2. Ethereum typically many validators per round (called an
+   [epoch](https://ethereum.org/en/glossary/#epoch) on Ethereum) to provide strong validity
+   guarantees while {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} can provide
+   stronger guarantees with fewer validators per round.
+   {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} achieves this by making validators
+   distribute an [erasure coding](./learn-parachains-protocol.md#erasure-codes) to all validators in
+   the system, such that anyone - not only the round's validators - can reconstruct a parachain's
+   block and test its validity. This data availability is a core part of Polkadot - ensuring state
+   is valid for its state transitions. The random parachain-validator assignments and secondary
+   checks are performed by randomly selected validators making it less likely for the small set of
+   validators on each parachain to collude.
+
+### Staking Mechanics: Ethereum PoS vs. Polkadot NPoS
 
 Ethereum is a proof-of-stake network that requires 32 ETH to stake for each validator instance.
 Validators run a primary Beacon Chain node and multiple validator clients - one for each 32 ETH.
-These validators get assigned to "committees," randomly selected groups to validate shards in the
-network. Ethereum relies on having a large validator set to provide availability and validity
-guarantees: They need at least 111 validators per shard to run the network and 256 validators per
-shard to finalize all shards within one epoch. With 64 shards, that's 16_384 validators (given 256
-validators per shard). See
-[Ethereum Economics](https://docs.ethhub.io/ethereum-roadmap/ethereum-2.0/eth-2.0-economics/) and
-[Eth2 shard chain simplification proposal](https://notes.ethereum.org/@vbuterin/HkiULaluS) for more
-information.
+These validators get assigned to "committees," randomly selected groups to validate blocks in the
+network.
 
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} can provide strong finality and
```

Comment:
I would delete the staking paragraph and add the following:

Polkadot uses [Nominated Proof of Stake (NPoS)](./learn-phragmen) to select validators from a smaller set, letting smaller holders nominate validators to run the network while claiming the system’s rewards without running a node. Polkadot needs about five validators for each parachain in the network. For more information, see the [staking page](./learn-staking).

Reply:
this is great!! exactly what I was really looking for, thanks!